hosting/photo-storage/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "photo-storage",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@dfinity/assets": "^0.13.4",
+    "@dfinity/agent": "^0.13.4",
+    "@dfinity/identity": "^0.13.4",
```

```suggestion
    "@dfinity/assets": "^0.14.0",
    "@dfinity/agent": "^0.14.0",
    "@dfinity/identity": "^0.14.0",
    "@dfinity/candid": "^0.14.0",
    "@dfinity/principal": "^0.14.0",
```
I had to do this to get it to work